exercises/concept/the-farm/the_farm_test.go

```diff
@@ -14,192 +14,141 @@ func (wf testWeightFodder) FodderAmount() (float64, error) {
 	return wf.fodder, wf.err
 }
 
+var NonScaleError = errors.New("non-scale error")
```

Comment:
This error was intenionally local to the test case as it is not relevant for other tests.

Reply:
Agree